packages/framer-motion/src/value/index.ts

```diff
@@ -223,12 +225,17 @@ export class MotionValue<V = any> {
      *
      * @public
      */
-    set(v: V, render = true) {
-        if (!render || !this.passiveEffect) {
-            this.updateAndNotify(v, render)
+    set(v: V, immediate = false) {
+        if (immediate || !this.passiveEffect) {
+            this.updateAndNotify(v)
```

Comment:
Yeah, and API-wise it does not _have_ to be a breaking change, right?
It's just nicer to be able to call it `immediate`.

I would suggest calling this new function something else, but that is a bit too disruptive I guess.

We could maybe as easy-backwards compatible version add a function that has the old behaviour?

Reply:
IMO this might be a breaking change in the sense that I stupidly documented the second argument (although it was never expanded upon) even though it wasn't meant to be public.

I don't think it's required to keep the existing old functionality as its usage was never particularly explained but I'd be happy to bump to `9.0.0`.

And the way we used it internally was always to achieve the same effect we're using adding the `immediate` flag for:

```
value.set(100, false)
value.set(100)

// velocity = 0 
```

The "don't render" flag was only really a means to this end.

Reply:
Thinking it through slightly more, just going to take a quick stab in a way that retains the existing functionality

Reply:
I think we should bump the version, albeit just for keeping semver